graph/src/firehose/endpoints.rs

```diff
@@ -22,14 +23,15 @@ use tonic::{
 
 use super::codec as firehose;
 
-const SUBGRAPHS_PER_CONN: usize = 100;
+pub const SUBGRAPHS_PER_CONN: usize = 100;
```

Comment:
What Eva and I had discussed was to make that control work the same as the one for RPC endpoints described [here](https://github.com/graphprotocol/graph-node/blob/master/docs/config.md#controlling-the-number-of-subgraphs-using-a-provider) to make it possible to control this for each node separately. (Implemented by [this](https://github.com/graphprotocol/graph-node/pull/3685) and [this](https://github.com/graphprotocol/graph-node/pull/3816) PR)

Since it seems this is only needed for some short-term testing, it's probably good enough for firehoses. But I think it should at least be settable with an environment variable; otherwise we need to change the code every time this limit needs to be adjusted.

Reply:
This is not really variable, it's a hard limit we found [here](https://github.com/graphprotocol/graph-node/issues/3879)

Reply:
Ok. It would be good to have a comment there then why this is a universal constant.

Reply:
Added the comment